src/bp/ui-studio/src/web/components/Layout/CommandPalette.tsx

```diff
@@ -78,7 +78,7 @@ const CommandPalette: FC<Props> = props => {
         url: window.location.origin + '/studio/' + bot.id
       })),
       ...props.modules
-        .filter(module => !module.noInterface)
+        .filter(module => !module.noInterface || module.name === 'nlu' || module.name === 'qna')
```

Comment:
leftover ?

Reply:
I wanted to be able to access nlu UI from command palette until we get everything implemented in flow builder. which is done now ?

Reply:
yup you can close this

Reply:
actually good point let's keep it until we remove the GUI entirely